setup.py

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env python3
-# this is a shim file to allow `python setup.py develop`/`pip install -e` to
-# work
 from setuptools import setup
+import re
+
+__version__ = re.search(
+    r'__version__\s*=\s*[\'"]([^\'"]*)[\'"]', open("skcosmo/__init__.py").read()
+).group(1)
```

Comment:
Is this the best way to do this? @Luthaf

Reply:
Same as your PR, this one is showing changes that are already in master. I don't know Why they show up here though

Reply:
Look like there is my commit inside the branch for this PR. @agoscinski did you use both merge and rebase in this branch?

Reply:
I am waiting for the preprocessing branch #8 before I rebase